src/clients/UBAClient/UBAClientAbstract.ts

```diff
@@ -142,23 +142,21 @@ export abstract class BaseUBAClient {
 
   /**
    * Compute the entire system fee for a given amount. The system fee is the sum of the LP fee and the balancing fee.
-   * @param depositChain The chainId of the deposit
-   * @param refundChain The chainId of the refund
+   * @param destinationChainId The chainId of the transaction
    * @param spokePoolToken The token to get the system fee for
    * @param amount The amount to get the system fee for
    * @param hubPoolBlockNumber The block number to get the system fee for
    * @returns The system fee for the given token on the given chainId at the given block number
    */
   public async computeSystemFee(
-    depositChain: number,
```

Comment:
Don't we still need the depositChainId? I'd thought that would be retained, but I might have misunderstood.

Reply:
I was of the understanding that this was different. Fair point though, I've updated it

Reply:
I also agree we need deposit chain ID

System Fee = deposit balancing fee + LP fee
- deposit balancing fee is based only on deposit chain ID
- LP fee is based on deposit chain and destination chain